### ISSUE TITLE
woltka classify parameter '--samples' doesn't work properly when supplied with a file (woltka, version 0.1.4, bug report)

### ISSUE THREAD

**Issue by ProsperP** [qiyunzhu/woltka#163]
Hi, I was trying to run `woltka classify` with `--samples metadata.txt` parameter (where metadata.txt is a list of sample ids along with the metadata). The program would always raise `ValueError('Duplicate IDs found.')`. So, I looked back into the source code according to the Traceback and found that at the 368 line of the `workflow.py` script uses `read_ids()` function to read a "sample list file" without opening it first (I've also noticed that `read_ids()` function only treats its input parameter as a `list-like` object). The source code:
```
samples = read_ids(samples) if isfile(samples) else samples.split(',')
```

And, I think this code might be modified to:
```
samples = read_ids(openzip(samples)) if isfile(samples) else samples.split(',')
```

Thanks for your attention.

**Comment by qiyunzhu** [comment]
@ProsperP Thank you for reporting this bug and proposing a solution. I have fixed it in #164 and merged to the master branch. It should work now. The program can be updated with:

```bash
pip install --force-reinstall --no-cache-dir git+https://github.com/qiyunzhu/woltka.git
```

**Comment by ProsperP** [comment]
@qiyunzhu Many thanks for your quick response and solution. It works now.
